master/getting-started/kubernetes/installation/config-options.md

```diff
@@ -57,6 +57,35 @@ want to disable IP-in-IP encapsulation, such as under the following circumstance
 To disable IP-in-IP encapsulation, modify the `CALICO_IPV4POOL_IPIP` section of the
 manifest.  For more information, see [Configuring {{site.nodecontainer}}]({{site.baseurl}}/{{page.version}}/reference/node/configuration).
 
+### Switching from IP-in-IP to VXLAN
```

Comment:
I worry that this wording makes the user think that this is an "upgrade" rather than what it is...  "Configuring VXLAN" might be a safer option.